node/hyperbahn/service_proxy.js

```diff
@@ -225,6 +225,10 @@ function createServiceChannel(serviceName) {
         options.requestDefaults = self.serviceReqDefaults[serviceName];
     }
 
+    if (mode === 'exit') {
+        options.preferConnectionDirection = 'out';
```

Comment:
remind me please, the default is now no-preference? that is what would make most sense for internal forwarding nodes imo.